system_tests/pubsub.py

```diff
@@ -246,5 +263,12 @@ def test_fetch_delete_subscription_w_deleted_topic(self):
                    if subscription.name == ORPHANED]
         self.assertEqual(len(created), 1)
         orphaned = created[0]
+
+        def _no_topic(instance):
+            return instance.topic is None
+
+        retry = RetryInstanceState(_no_topic)
+        retry(orphaned.reload)()
```

Comment:
I know, but you can roll your own method that returns the instance (like I just did) without having to carry around an extra class.